Excel_UI/Ribbon/Ribbon.cs

```diff
@@ -51,7 +51,7 @@ public override string GetCustomUI(string RibbonID)
             <group id='ui' label='UI'>
                 <button id='expand' onAction='RunExpand' label='Expand list' imageMso='OutlineExpandAll' supertip='Take a list stored in a single cell and expand over multiple cells (one cell per item in the list).'/>
                 <button id='condense' onAction='RunCondense' label='Condense cells' imageMso='CollapseAll' supertip='Take a group of cells and store their content as a list in a single cell.'/>
-                <button id='internalise' onAction='Internalise' label='Internalise data' imageMso='RecordsSaveRecord' supertip='Save the value of the selected cells so it will stil lbe available when the file is reopened.' />
+                <button id='internalise' onAction='Internalise' label='Internalise data' imageMso='RecordsSaveRecord' supertip='Save the value of the selected cells so it will still be available when the file is reopened.' />
```

Comment:
```suggestion
                <button id='internalise' onAction='Internalise' label='Internalise data' imageMso='RecordsSaveRecord' supertip='Save the value of the selected cells so that the values will be available when the file is reopened.' />
```

I think this reads slightly better given the function of the action? Thoughts @dengh0 ?

Reply:
Yeah I agree!